cnf-certification-test/certification/suite.go

```diff
@@ -202,3 +209,33 @@ func testContainerCertificationStatusByDigest(env *provider.TestEnvironment, val
 	}
 	testhelper.AddTestResultLog("Non-compliant", failedContainers, tnf.ClaimFilePrintf, ginkgo.Fail)
 }
+
+func testHelmVersion(env *provider.TestEnvironment) {
+	var compliantObjects []*testhelper.ReportObject
+	var nonCompliantObjects []*testhelper.ReportObject
+	tillerFound := false
+	clients := clientsholder.GetClientsHolder()
+	for _, tillerNamespace := range env.AllNamespaces {
+		// Get the Tiller pod in the specified namespace
+		podList, err := clients.K8sClient.CoreV1().Pods(tillerNamespace).List(context.TODO(), metav1.ListOptions{
```

Comment:
remove the namespace loop and use .Pods("") to loop through all namespaces
podList, err := clients.K8sClient.CoreV1().Pods("").List(context.TODO(), metav1.ListOptions ...